clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -622,7 +618,7 @@ void abortBatches(final RuntimeException reason) {
         }
     }
 
-    void abortOpenBatches(RuntimeException reason) {
+    void abortUnsentBatches(RuntimeException reason) {
```

Comment:
I renamed it to try and be consistent with `hasUnsent` and `hasIncomplete`. I think perhaps changing it to `abortUndrainedBatches` and `hasUndrained` is reasonable.